catalogs/catalog-lakehouse-paimon/src/main/java/org/apache/gravitino/catalog/lakehouse/paimon/PaimonCatalogPropertiesMetadata.java

```diff
@@ -46,14 +46,26 @@ public class PaimonCatalogPropertiesMetadata extends BaseCatalogPropertiesMetada
   public static final String PAIMON_METASTORE = "metastore";
   public static final String WAREHOUSE = "warehouse";
   public static final String URI = "uri";
+  public static final String JDBC_USER = "jdbc.user";
```

Comment:
could you use `jdbc-user` like Iceberg?

Reply:
Paimon use `jdbc.` as property prefix.

Reply:
Iceberg internally uses `jdbc.user` too, we'd better provide a unified Gravitino Jdbc username.

Reply:
updated. @FANNG1